Automattic/wp-calypso
# Fetch whether a domain supports GDPR consent management from the back end.

#### Changes proposed in this Pull Request

* Retrieve whether a domain supports GDPR consent management from the back end.
* Use this information to determine whether to show the link to the consent management page instead of checking the domain's registry.

#### Testing instructions

* Depends on D22849-code
* See the differential for instructions on how to generate new test domains.
* Visit the contacts privacy page for various domains.
* Make sure that the link to the consent management page is only shown for domains that support this feature.
